ZEP/instructions/zep0000.md

```diff
@@ -0,0 +1,201 @@
+`Accepted`.
+
+Once a ZEP has been `Accepted`, the reference implementation must be completed. When the reference implementation is complete and incorporated into the main source 
+code repository, the status will be changed to `Final`.
+
+To allow gathering of additional design and interface feedback before committing to long term stability for a feature or standard library API, ZEP may also be 
+marked as `“Provisional”`. This is short for “Provisionally Accepted”, and indicates that the proposal has been accepted for inclusion in the reference 
+implementation, but additional user feedback is needed before the full design can be considered “Final”. Unlike regular accepted ZEPs, provisionally accepted 
+ZEPs may still be Rejected or Withdrawn even after the related changes have been included in a Zarr release.
+
+Wherever possible, it is considered preferable to reduce the scope of a proposal to avoid the need to rely on the `“Provisional”` status (e.g. by deferring some features to later ZEPs), as this status can lead to version compatibility challenges in the wider Zarr ecosystem.
+
+A ZEP can also be assigned status `Deferred`. The ZEP author or [Zarr Steering Council](https://github.com/zarr-developers/governance/blob/master/GOVERNANCE.md#steering-council) can assign the ZEP this status when no progress is being made on the ZEP.
+
+A ZEP can also be `Rejected`. Perhaps, after all, is said and done it was not a good idea. It is still important to have a record of this fact. The `Withdrawn` 
+status is similar—it means that the ZEP author themselves has decided that the ZEP is a bad idea, or has accepted that a competing proposal is a better alternative.
+
+When a ZEP is `Accepted`, `Rejected`, or `Withdrawn`, the ZEP should be updated accordingly. In addition to updating the status field, at the very least the 
+`Resolution` header should be added with a link to the relevant link of the discussion.
+
+ZEPs can also be `Superseded` by a different ZEP, rendering the original obsolete. The `Replaced-By` and `Replaces` headers should be added to the original and 
+new ZEPs respectively.
+Process ZEPs may also have a status of `Active` if they are never meant to be completed, e.g. ZEP 0 (this ZEP).
+
+### How does a ZEP becomes accepted?
+
+A ZEP is `Accepted` by the consensus of all interested contributors. We need a concrete way to tell whether consensus has been reached. When you think ZEP is ready to accept, create a new discussion in [zarr-python](https://github.com/zarr-developers/zarr-python/discussions) using ‘General’ as the category with a subject like:
+
+> `Proposal to accept ZEP <number>: <title>`
+
+In the body of your discussion, you should:
+
+- Link to the latest version of ZEP,
+- Briefly describe any major points of contention and how they were resolved,
+- Include a sentence like: “If there are no substantive objections within 7 days from this post, then the ZEP will be accepted;
```

Comment:
Slightly concerned about the "silence == acceptance" route.

Reply:
I think some sort of quorum rule is useful here. We're looking at (and have been loosely using) [Martha's rules](https://third-bit.com/files/2020/08/marthas/) for scverse, which says at least half the voting members had to hold a vote for acceptance.

Reply:
> Slightly concerned about the "silence == acceptance" route.

if you think increasing the timeframe would help then we could do that too but I believe having a fixed timeframe for the community to raise objections would help us in preventing open-ended discussion and reaching conclusions.

May I know what concerns you about "silence"?

Reply:
> We're looking at (and have been loosely using) [Martha's rules](https://third-bit.com/files/2020/08/marthas/) for scverse, which says at least half the voting members had to hold a vote for acceptance.

This is interesting. Thanks for sharing @ivirshup.

Reply:
> May I know what concerns you about "silence"?

It's hard to know if people are not disagreeing simply because they don't have time to respond.

Reply:
Added a few lines to tackle this [here](https://github.com/MSanKeys963/governance/blob/master/ZEP/instructions/zep0000.md#how-does-a-zep-becomes-accepted).